src/IssueViz.Security/IssueViz.Security.csproj

```diff
@@ -64,6 +75,7 @@
       <Generator>XamlIntelliSenseFileGenerator</Generator>
       <CopyToOutputDirectory>Never</CopyToOutputDirectory>
     </Page>
+    <Resource Include="Taint\TaintList\sonarlint.png" />
```

Comment:
Project files are easier to understand if related changes are grouped together e.g. group removing this from the `None` group with adding it to the `Resource` group.

Sometimes it makes sense to use a separate `ItemGroup` with an explanatory label e.g.

```
<ItemGroup Label="XAML resources">
  <None Remove="Taint\TaintList\sonarlint.png" />
  <Resource Include="Taint\TaintList\sonarlint.png" />
</ItemGroup>
```